package.json

```
@@ -39,7 +39,7 @@
     "stylelint-config-standard": "^20.0.0",
     "tailwindcss": "^2.0.3"
   },
-  "dependencies": {
-    "jquery": "^3.5.1"
+  "peerDependencies": {
+    "jquery": "3.5.1"
```

I expected the version range to still hold true 

```suggestion
    "jquery": "^3.5.1"
```

---

Do they keep it updated like that?